testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -478,6 +482,15 @@ class BaseDistanceSelection(object):
 
     Cylindrical methods don't use KDTree
     """
+
+    __test__ = False
+
+    @staticmethod
+    @pytest.fixture()
+    def u(self):
+        # TODO: This is dummy. Needs review!
```

Comment:
Now that we don't need to pass universe to the parametrize call, we can get rid of this completely.